### ISSUE TITLE
elementStyle: Returning element for conditional sizes.

### ISSUE THREAD

**Issue by igoriok1994** [nathancahill/Split.js#149]
Hi there,

Need a possibility to access element, for conditional sizing (in elementStyle function)
Functions has 'dimension, elementSize, gutterSize' arguments.
Need additional argument el (element)

![image](https://user-images.githubusercontent.com/37294916/45676216-e851b900-bb39-11e8-8bda-4e452dc4e280.png)

**Comment by igoriok1994** [comment]
Yes, but I have to know elements name or id, to return good size.
For example:
I have 2 elements, and gutter between.
For first element I have to return calc(elementSize + '% - 30px)
For second - calc(elementSize + '% - 55px)
But I have to know the element name or id or something (then i can define good size).
The problem is than i don't know with which element i working, and to which element this size will be applied

**Comment by nathancahill** [comment]
Published in v1.5.4.
